app.js

```diff
@@ -72,12 +79,19 @@ const download = (newspaper, date) => {
   const name = `${newspaper.name} for ${date}`
 
   if (fs.existsSync(pdfPath)) {
-    return fs.existsSync(pngPath) ? Promise.resolve(log.debug(`Already downloaded ${name}`)) : pdfToImage(pdfPath, pngPath)
+    return fs.existsSync(pngPath) ?
+        Promise.resolve(
+            config.verbose ?
```

Comment:
Do we really need config.verbose? Its used only once. We could set the log level to `log.trace` here if you wish

Reply:
Updated.

Reply:
`console.trace` is not a good option here, as it outputs a stack trace: https://developer.mozilla.org/en-US/docs/Web/API/console/trace